CHANGELOG.md

```diff
@@ -17,6 +17,11 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [6.33.2] - 2023-10-16
+### Fixed
+- When fetching datapoints from "a few time series" (implementation detail), all missing, non-ignorable time series
+  are now raised together in a `CogniteNotFoundError` rather than only the first encountered.
+
```

Comment:
```suggestion
## [6.33.2] - 2023-10-16
### Fixed
- When fetching datapoints from "a few time series" (implementation detail), all missing, non-ignorable time series
  are now raised together in a `CogniteNotFoundError` rather than only the first encountered.

### Improved
- Datapoints fetching has a lower peak memory consumption when fetching from multiple time series simultaneously.
```